elastic/kibana
# [Fleet] Fix displaying agent status bar with no agents

## Summary

Resolve #121483

Fix displaying the agent status bar when filtering results in no agents.

<img width="1051" alt="Screenshot 2023-05-03 at 9 19 36 AM" src="https://user-images.githubusercontent.com/1336873/235930416-380e3936-2652-4deb-bd1c-5e8f213c3928.png">


## How to test

1. Add some agents to Fleet
2. Go to the list page and disable some status you have no agents => the status bar should not be visible anymore
